.docker/ci/Dockerfile

```diff
@@ -1,50 +1,90 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit2:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+LABEL maintainer="mike@picknik.ai"
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/opt/ws_moveit
 
-# Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
-# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
-RUN \
-    mkdir src && \
-    cd src && \
-    #
-    # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
-    #
-    # Update apt package list as previous containers clear the cache
-    apt-get -qq update && \
-    apt-get -qq dist-upgrade && \
+WORKDIR $ROS_WS
+
+# Update apt package list as previous containers clear the cache
+RUN apt-get -qq update && \
```

Comment:
I think these RUN commands should be combined again eventually.

Reply:
But that would negatively impact layer reuse/caching (see above).